src/crewai/tools/tool_usage.py

```diff
@@ -11,11 +11,10 @@
 from crewai.utilities import I18N, Converter, ConverterError, Printer
 
 agentops = None
-if os.environ.get("AGENTOPS_API_KEY"):
```

Comment:
shouldn't we keep this? given that if the person haven't set up a key we probably don't was to try import the library anyway?